contents/javascript/Javascript-framework-Angular.md

```diff
@@ -0,0 +1,202 @@
+<frontmatter>
+  title: Introduction to Angular
+  footer: footer.md
+  head: head.md
+  siteNav: mainNav.md
+  pageNav: 3
+</frontmatter>
+
+{{ navbar | safe }}
+
+<div class="website-content">
+
+# Introduction to Angular
+
+Author(s): [Ronak Lakhotia](https://github.com/RonakLakhotia)
+
+## What Is Angular?
+
+Angular is an open-source frontend Javascript framework that helps you build <tooltip content="Web apps that are more responsive to user interactions ">Single Page Web Applications</tooltip>.
+The framework is designed to better engineer applications in an effort to make development and testing easier for enterprise web applications. Angular is maintained by Google, as well as a community of individual developers.
+
+>Angular started as a side project. Back in 2009, Miško Hevery and Adam Abrons released a project that would help developers, as well as designers, build web applications using simple HTML tags. The name “Angular” comes from the angle brackets, or < >, that surround all HTML tags.
+>
+>Miško described the idea behind the framework in an <a href="https://www.infoworld.com/article/2612801/javascript/whats-so-special-about-googles-angularjs.html">interview done in 2013</a>. 
+
+
+## Why learn Angular?
+
+Angular is not the world’s simplest framework, and it takes time to understand some of the concepts that Angular has to offer.
```

Comment:
There are two Google hits for the exact phrase `"Angular is not the world’s simplest framework, and it takes time to understand some of the concepts that Angular"` I Googled for it because it felt like lifted from somewhere else when I read it.